### ISSUE TITLE
Declaration of Conformance

### ISSUE THREAD

**Issue by greg-westnine** [eQualityTime/TheOpenVoiceFactory#89]
The legal document which asserts that the Open Voice Factory product is a class I medical device which complies with the relevant regulation.

**Comment by joereddington** [comment]
Next action is to write our our own form of words and then put it in a document.

**Comment by joereddington-public** [comment]
> "The content of the Declaration of Conformity is described in GHTF/SG1/N40:2006 Principles of Conformity Assessment for Medical Devices."

**Comment by joereddington-public** [comment]
Our form of words 

>I, Joseph Reddington, Chief opperating office of eQuality Time declare that the Open Voice Factory software, which is hosted at http://www.theopenvoicefactory.org/ complies with the applicable Essential principles for Safety and Performance, has been classified as a class 1 device acccording to the classification rules and has met all the applicable conformity assessment elements.   

I need the GMDN code [this page](https://accessgudid.nlm.nih.gov/devices/07340074600304) suggests that it is "Voice synthesizer/digitizer", and that's good enought for me, but I've also registered as an academic research on the GMDN website as well.  

> The Open Voice Factory's GMDN code is "Voice synthesizer/digitizer".  It is of Risk Class A according to Rule 12 of the Principles of Medical Devices Classification"

@greg-westnine - I have no  idea about "which of the conformity assessment elements have been applied... fill me in?

**Comment by greg-westnine** [comment]
I agree that with the suggestions that OVF is a 'voice synthesizer/digitizer' but I think Risk Class A, rule 12 is referring to FDA rules, not EU.  We can quote the assessment in the Device Description that OVF is a Class I device.
I think conformity assessment elements means the evidence you are providing, ie the contents of the technical file.

**Comment by greg-westnine** [comment]
Updated to be more in line with some examples I reviewed, and also to fill in the checklist more thoroughly.
[Declaration of Conformity.docx](https://github.com/eQualityTime/TheOpenVoiceFactory/files/2447300/Declaration.of.Conformity.docx)
